Signal/src/call/PeerConnectionClient.swift

```diff
@@ -697,6 +718,14 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
             assert(self.dataChannel == nil)
             self.dataChannel = dataChannel
             dataChannel.delegate = self
+
+            let pendingMessages = self.pendingDataChannelMessages
+            self.pendingDataChannelMessages = []
+            DispatchQueue.main.async {
```

Comment:
Shouldn't we capture weakself and unpack it to strongself here?

Reply:
Yes! Fixed.